provider/mediaconvert/vp8.go

```diff
@@ -0,0 +1,51 @@
+package mediaconvert
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/mediaconvert"
+	"github.com/cbsinteractive/transcode-orchestrator/db"
+	"github.com/pkg/errors"
+)
+
+const (
+	defaultGopUnitVP8 = "frames"
+)
+
+func vp8CodecSettingsFrom(preset db.Preset) (*mediaconvert.VideoCodecSettings, error) {
+	if preset.Video.GopUnit != defaultGopUnitVP8 {
```

Comment:
We default to `frames` if the `GopUnit` is an empty string here: https://github.com/cbsinteractive/transcode-orchestrator/blob/master/provider/mediaconvert/h264.go#L66

perhaps we should update this to 
```suggestion
	if u := preset.Video.GopUnit; len(u) > 0 && u != defaultGopUnitVP8 {

Reply:
That default is only for h264 it seems, though Im sure h265 has that same implementation. I wasn't planning on introducing that here as we wouldn't want to default. we just want to throw an error.

Reply:
yeah, we do the same for https://github.com/cbsinteractive/transcode-orchestrator/blob/master/provider/mediaconvert/h265.go#L79 I think this speaks to overall usability and think it's worthwhile to simply default to frames if no value is passed. If anything is passed that is _not_ frames, that seems like the right time to error.